index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>mobile version</title>
+</head>
+<body>
+  <header class="header">
+      <a href="index.html">Arrow Dan</a>
+      <nav class="navbar container">
+        <img class="hamburger" src="image/Union.png" alt="navigation icon">
+      </nav>
+  </header>
+
+  <section class="main-container">
+
+  <section class="corner-image">
+    <div class="container">
+        <h1 class="logo-message">I'm  Dan</h1>
+        <h3 class="dev-intro">I'am a Software Developer</h3>
+        <p class="background">
+            I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, do not hestiate to contact me.
+        </p>
+    </div>
+    <ul class="social-media  container">
+        <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
+        <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
+        <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
+        <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
+        <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
```

Comment:
- [ ] [OPTIONAL] Please remove the horizontal scroll in the page as it does not give a good user experience.